src/github/queries.js

```diff
@@ -128,11 +141,27 @@ const organizationRepositories = (login, pagination) => {
   return getOrganizationData(inputs, data, variables);
 };
 
+const totalPROrganization = (login) => {
+  const query = `org:${login} type:pr`;
+  const data = "issueCount";
+
+  return getSearchData(query, "ISSUE", data);
+}
+
+const totalIssuesOrganization = (login) => {
+  const query = `org:${login} type:issue`;
+  const data = "issueCount";
+
+  return getSearchData(query, "ISSUE", data);
+}
+
 export default {
   me,
   organization,
   organizationTeams,
   organizationMembers,
   organizationRepositories,
   organizationTeamMembers,
+  totalPROrganization,
```

Comment:
I think this and the next line should be following the naming pattern from variables above. organization**PropName** just like in the api.js and expectedResult.js file that was like `organizationTotalPullRequests`